orderable/tests/test_models.py

```diff
@@ -207,3 +213,13 @@ def test_changing_parent(self):
         subtask_2.task = task
         subtask_2.save()
         self.assertSequenceEqual(task.subtask_set.all(), [subtask, subtask_2])
+
+    @given(lists(integers(min_value=1), min_size=1, unique=True))
```

Comment:
I've read the Hypothesis docs before, last time we were talking about using it.  It looked a bit rubbish to me at the time.  I'd appreciate an explanation of why we're suddenly using it and what its benefits are, please.